client/windows/CHeroWindow.cpp

```diff
@@ -103,33 +102,37 @@ CHeroWindow::CHeroWindow(const CGHeroInstance *hero):
 	//artifs = new CArtifactsOfHero(pos.topLeft(), true);
 	ourBar = new CGStatusBar(7, 559, "ADROLLVR.bmp", 660); // new CStatusBar(pos.x+72, pos.y+567, "ADROLLVR.bmp", 660);
 
-	quitButton = new CButton(Point(609, 516), "hsbtns.def", CButton::tooltip(heroscrn[17]), [&]{ close(); }, SDLK_RETURN);
+	quitButton = new CButton(Point(609, 516), "hsbtns.def", CButton::tooltip(heroscrn[17]), [&] { close();
+	}, SDLK_RETURN);

```

Comment:
@mwu-tow this will be fixed once #345 merged. Sorry that I didn't mention it earlier.
Basically Uncrustify have a hard time to parse lambdas with omitted parameter list.